subsys/bluetooth/shell/bap_broadcast_assistant.c

```diff
@@ -16,12 +16,12 @@
 #include <zephyr/bluetooth/gatt.h>
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/hci.h>
-#include <zephyr/bluetooth/audio/bass.h>
+#include <zephyr/bluetooth/audio/bap.h>
```

Comment:
Since everywhere else it is updated the Copyright year should include 2022 here as well